src/silo/test/randomize.test.cpp

```diff
@@ -0,0 +1,85 @@
+#include <nlohmann/json.hpp>
+
+#include <optional>
+
+#include "silo/test/query_fixture.test.h"
+
+using silo::ReferenceGenomes;
+using silo::config::DatabaseConfig;
+using silo::config::ValueType;
+using silo::test::QueryTestData;
+using silo::test::QueryTestScenario;
+
+const std::vector<nlohmann::json> DATA = {
+   {{"metadata", {{"key", "id1"}, {"col", "A"}}},
```

Comment:
Do we need "col"? If not, I think it's better to keep the datasets as minimal as possible.

```suggestion
   {{"metadata", {{"key", "id1"}}},
```

Reply:
Added it for a sort test, but forgot that implementation hehe